src/createClient.js

```diff
@@ -86,15 +114,29 @@ function connect (client) {
       clearInterval(keepalive)
     })
   }
-}
 
-async function ping ({ host, port }) {
-  const con = new RakClient({ host, port })
+  client.once('close', () => {
+    if (client.session) client.session.end()
+    if (client.signalling) client.signalling.destroy()
```

Comment:
Is client.session not something already auth related?

Reply:
Yeah it is, I took note of that a while back to amend.

Reply:
Putting all the nethernet properties under .nethernet.* may be a good idea